config/webpack.config.base.js

```diff
@@ -26,10 +26,7 @@ module.exports = {
     rules: [
       {
         test: /\.jsx?$/,
-        include: [
-          path.resolve(__dirname, '../src'),
-          path.dirname(require.resolve('cozy-ui'))
-        ],
+        exclude: /node_modules\/(!?(cozy-ui))/,
```

Comment:
Should we be more precise with a `cozy-ui/react` here?

Reply:
I guess we should transpile everything in cozy-ui, not only the react folder.

Reply:
Why ? cozy-ui/react/transpiled is already transpiled.

Reply:
@ptbrowne you mean `cozy-ui/transpiled/react` I think?